src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java

```diff
@@ -75,7 +78,12 @@ protected EmployeeDto toDTO(Employee entity) {
 
         List<Contract> contracts = contractRepository.findAllByEmployeeIdAndDeletedIsFalse(entity.getId());
         Optional<Contract> firstContract = contracts.stream().min(Comparator.comparing(Contract::getStartDate));
-        return new EmployeeDto(entity, paymentInformationList, lastAssignment.orElse(null), firstContract.orElse(null));
+        String availableDays = "0";
+        boolean employeeHasVacations = vacationRepository.existsVacationsByEmployee_IdAndDeletedIsFalse(entity.getId());
+        if(employeeHasVacations){
+            availableDays = Integer.toString(vacationRepository.getAvailableDays(entity.getId()));
+        }
+        return new EmployeeDto(entity, paymentInformationList, lastAssignment.orElse(null), firstContract.orElse(null), availableDays);
```

Comment:
Here you should only call to `vacationRepository.getAvailableDays(entity.getId())`. Then you can remove the check with `existsVacationsByEmployee_IdAndDeletedIsFalse`

Also, why not return an integer?